docs/includes/apiargs-MongoDBCollection-method-createIndex-option.yaml

```diff
@@ -1,4 +1,23 @@
 arg_name: option
+name: commitQuorum
+type: string|integer
+description: |
+  Specifies how many data-bearing members of a replica set, including the
+  primary, must complete the index builds successfully before the primary marks
+  the indexes as ready.
+
+  This option accepts the same values for the "w" field in a write concern plus
+  "votingMembers", which indicates all voting data-bearing nodes.
```

Comment:
I'd revise as:

> ... in a write concern in addition to the string `"votingMembers"`, which...

Monospace formatting helps indicate that we're talking about a string value here. I think we do that elsewhere in our docs.

The "w" field could probably just be reformatted as `w` (monospace, but no quotes), since we're only referring to a key.